src/main/webapp/i18n/en/exercise.json

```diff
@@ -187,7 +187,10 @@
                     "instructionDeleted": "Grading Instruction deleted:",
                     "scoringChange": "Scoring changed:",
                     "results": "The existing results must be recalculated!",
-                    "confirmtext": "Do you really want to perform these changes?"
+                    "confirmtext": "Do you really want to perform these changes?",
+                    "reevaluate": "Re-evaluate",
+                    "deleteFeedbacks": "Additionally delete all corresponding feedbacks that are used to assess submissions.",
+                    "information": "By clicking 'Re-evaluate', the assessments will be re-evaluated and the exercise will be updated. If you do not want to change existing assessments, you can click 'Confirm Changes' and you have to accept the consequences. This will only update the exercise and grading instructions. Therefore, it can cause inconsistencies."
```

Comment:
Acutally, I do not understand this explanation. If I do not want to change existing assessments, I have to click 'Confirm Changes'? This does not seem intuitive to me, but maybe I don't understand this. I think the inconsistencies mentioned should also be explained, which inconsistencies?

Reply:
hmm, well we leave this decision up to Instructor. In case there are changes, that would lead to inconsistencies (e.g. change the score, delete a SGI), we have a two option: It updates the exercise and SGI after that there will be inconsistencies. Let's say he/she change the score of one of the grading instruction which is already associated with feedback. So, if you only update exercise. when you look at the feedback table, you will see feedback score and associated grading instruction credit are not same. I meant this kind of inconsistencies. After updating grading instructions, there may be a time when they don't want to change the existing feedback used during assessment period. So, the Instructor will be aware of the situation, can decide to re-evaluate or not. Maybe, I did not explain the situation well in this `information` part. Is it more clear now?

Reply:
Ok, I think I understand now... I tested in also and saw which inconsistencies were meant. Following suggestion:

<img width="1545" alt="Bildschirmfoto 2021-06-08 um 17 01 47" src="https://user-images.githubusercontent.com/41366522/121213769-d29ef780-c87e-11eb-8ddc-779d2f723213.png">

I also thought that the structure of the warning is a bit overwhelming so I propose text and structure with this picture ;)

Reply:
ohhh, I see. it looks better... In case if the user deletes grading instructor, I will add only bold Instruction Deleted text in here, and put the checkbox to delete corresponding feedback. Is that okay? After that, i can update the translation files.

Reply:
applied the changes, tried to translate them. you can quickly check that 😊